recipes/torchsparse/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "torchsparse" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mit-han-lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 6fa80a875eb2bfa74a69dd1b8e64df1b699710d07c072ca504d4f25fbbcb8730
+
+build:
+  number: 0
+  skip: true                                 # [osx or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+  missing_dso_whitelist:
+    - "$RPATH/libc10.so"
+    - "$RPATH/libtorch_cpu.so"
+    - "$RPATH/libtorch_python.so"
+    - /lib/libstdc++.so.6
+    - /lib*/libgcc_s.so.1
+    - /lib*/libc.so.6
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - pybind11                               # [build_platform != target_platform]
```

Comment:
How come these three are only needed when cross compiling?

Reply:
I was prepping it for https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock -- maybe that's not needed, though?

Reply:
It’s probably easiest to add it to the migrator (see the first paragraph)